packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -68,6 +68,7 @@ export class AtomicSearchInterface
   @Element() public host!: HTMLAtomicSearchInterfaceElement;
 
   @State() public error?: Error;
+  @State() openRelevanceInspector = false;
```

Comment:
the verb `open` makes it sound to me like this is a method as opposed to a state. I think we should rename this to something like `relevanceInspectorIsOpen`.